shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java

```diff
@@ -250,4 +258,63 @@ private OptimizerContext createOptimizerContext() {
         when(result.getFederationMetaData().getSchemas()).thenReturn(schemas);
         return result;
     }
+    
+    @Test
+    public void assertRenewPrimaryDSNames() {
```

Comment:
1. `DS` is abbreviate, could you use `DataSource` instead of it?
2. ShardingSphere only can support single primary data source, should the method name rename from `Names` to `Name`?

Reply:
+1. I'll update the code